admin/index_m.html

```diff
@@ -393,22 +318,23 @@
                             <label for="gwip" class="translate">Gateway IP</label>
                         </div>
                         <div class="col s6 l1 input-field">
-                            <input type="number" min="1025" max="65535" class="value" id="gwipport" />
+                            <input type="number" min="0" max="65535" class="value" id="gwipport" />
                             <label for="gwipport" class="translate">Port</label>
                         </div>
-                        <div class="col s6 l1 input-field">
-                            <input id="frameInterval" class="value" type="number" min="20" max="1000" />
-                            <label for="frameInterval" class="translate">Frames per second</label>
+                        
+                        <div class="col s12 l3 input-field">
+                            <input class="value" id="eibadr" type="text" />
+                            <label for="eibadr" class="translate">Physical KNX address</label>
                         </div>
                     </div>
                     <div class="row">
                         <div class="col s12 l4 input-field">
-                            <select class="value" id="bind"></select>
-                            <label class="translate" for="bind">local network interface</label>
+                            <select class="value" id="localInterface"></select>
+                            <label class="translate" for="localInterface">Local IPv4 network interface</label>
                         </div>
-                        <div class="col s12 l2 input-field">
-                            <input class="value" id="eibadr" type="text" />
-                            <label for="eibadr" class="translate">physical KNX address</label>
+                        <div class="col s6 l2 input-field">
+                            <input id="minimumDelay" class="value" type="number" min="0" max="1000" />
+                            <label for="minimumDelay" class="translate">Frames per second</label>
```

Comment:
minimum Delay = Frames per second ?
minimum Delay of 10sec = 6 Frames per second
10 Frames per seconds = minimum delay of 6 sec